doc/articles/features/windows-ui-notifications.md

```diff
@@ -0,0 +1,23 @@
+# Uno Support for Windows.UI.Notifications
+
+## Badge notifications
+
+Badge notifications are supported on iOS and macOS.
+
+macOS supports numeric and textual badges (as opposed to UWP) but does not support glyphs.
+
+iOS allows for numeric badges only. It is also necessary to request user permission prior to executing the badge notification for the first time (this request can be executed when appropriate for your application):
+
+```
+#if __IOS__
+UNUserNotificationCenter.Current
```

Comment:
Is there a reason why this cannot be included in the BadgeUpdateManager ?

Reply:
It theoretically could, but then the permission request could appear all of a sudden, out of context (especially if the badge is set in response to some syncing or data update check). This way the developer can show it when appropriate and later update the badge when required. In addition the Update and Clear methods would finish asynchronously (while the developer would assumed it has already been set).